### PR TITLE
Add CF and Sec frameworks on Mac to package_info

### PR DESCRIPTION
**Pull request by biewers**
I would like to make this change for Macs. It is required for the client side. I considered adding options for this. Please let me know if you would like me to alter this.

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -147,3 +147,6 @@ def package_info(self):
             self.cpp_info.libs.extend(["winhttp", "httpapi", "bcrypt"])
         if not self.options.shared:
             self.cpp_info.defines.append("_NO_ASYNCRTIMP")
+        if self.settings.os == "Macos":
+            self.cpp_info.exelinkflags.append("-framework CoreFoundation")
+            self.cpp_info.exelinkflags.append("-framework Security")
```
Comment (SSE4) [inline conanfile.py]: they also required to be added to `self.cpp_info.sharedlinkflags`

**Review by SSE4 — CHANGES_REQUESTED** [review]
LGTM

File: conanfile.py (hunk 1 above)
Comment (biewers) [reply conanfile.py]: Done

**Review by SSE4 — APPROVED** [review]
LGTM

**Pull request merged**